src/borda_clj/client.clj

```diff
@@ -21,6 +21,13 @@
         (do (print "borda buffer full, discarding measurement" dimensions values)
             (update measurements discard_key (partial merge-with +) {:success_count 1})))))) ; buffer full
 
+(defn merge-global [global-dimensions kv]
+  (let [[dimensions values] kv]
```

Comment:
You could have destructured `dimensions` and `values` in the function parameter list directly:

```clojure
(defn merge-global [global-dimensions [dimensions values]]
  ...)
```

Reply:
Yeah, figured that out :)